### PR TITLE
tests: rst linter added into travis tests

### PR DESCRIPTION
**Pull request by pwilczynskiclearcode**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,7 +13,8 @@ install:
     - "pip install wheel"
 script:
     - "py.test --verbose --capture=no --showlocals --cov mirakuru"
-    - pylama
+    - "pylama"
+    - "rst-lint README.rst"
```
Comment (fizyk) [inline .travis.yml]: also CHANGES.rst

**Comment by fizyk** [comment]
I see new pylama has been released ;)

**Comment by pwilczynskiclearcode** [comment]
> I see new pylama has been released ;)

I've fixed them in other pull request.

**Comment by pwilczynskiclearcode** [comment]
Pygments seems not to work on python 3.2 https://travis-ci.org/ClearcodeHQ/mirakuru/jobs/57193679

**Comment by fizyk** [comment]
Okay then we should either:

* drop support for py3.2
* use specific version of pygments
* switch off rest linter on python3.2 builds

**Comment by pwilczynskiclearcode** [comment]
I would switch off rest linter on python 3.2 builds.

**Comment by fizyk** [comment]
Let's do this then

**Comment by pwilczynskiclearcode** [comment]
```
script:
     - "py.test --verbose --capture=no --showlocals --cov mirakuru"
    - "pylama"
    - "PYTHON_V=`python -c 'import sys; print("".join(map(str, sys.version_info[:2])))'`
       if [ $PYTHON_V -eq 32 ]; then rst-lint README.rst; fi"
```

**Comment by pwilczynskiclearcode** [comment]
Greg? @fizyk

**Comment by pwilczynskiclearcode** [comment]
Well after a consideration I could use [TravisCI matrix](http://docs.travis-ci.com/user/build-configuration/#Excluding-jobs-with-a-shorthand) instead. so hold on :D

File: CHANGES.rst [hunk 2]
```diff
@@ -2,7 +2,7 @@ CHANGELOG
 =========
 
 unreleased
--------
+----------
```
Comment (fizyk) [inline CHANGES.rst]: Okay, this should also be changed in .bumpversion.cfg

File: .bumpversion.cfg [hunk 3]
```diff
@@ -10,7 +10,6 @@ current_version = 0.4.0
 
 [bumpversion:file:CHANGES.rst]
 search = unreleased
-	-------
+	----------
 replace = {new_version}
-	-------
-
+	----------
```
Comment (fizyk) [inline .bumpversion.cfg]: wouldn't that get too long for new version for the rstlinter?

**Comment by fizyk** [comment]
:+1: 

Just check if the linter will accept the underline longer than the version number in CHANGES.rst

**Comment by pwilczynskiclearcode** [comment]
Any longer seems to be OK.

**Pull request merged**